ObjectiveGit/GTRepository+Status.h

```diff
@@ -57,8 +59,8 @@ extern NSString *const GTRepositoryStatusOptionsFlagsKey;
 /// An enum, for use as documented, with the `GTRepositoryStatusOptionsFlagsKey`
 /// key.
 ///
-/// See status.h for documentation of each individual flag.
-typedef enum {
+/// See status.h for documentation of each individual git_status_opt_t flag.
+typedef NS_OPTIONS(NSInteger, GTRepositoryStatusFlags) {
```

Comment:
Well, `NS_ENUM` implies a single enumeration whereas `NS_OPTIONS` allow multiple values, especially when it bridges stuff to Swift, so you can do stuff like `[ .includeUntracked, .includeIgnored ]` natively. By the meaning of the enumeration, it's really an options bitfield, hence the choice of `NS_OPTIONS` over `NS_ENUM`.